modhub.php

```diff
@@ -75,7 +75,7 @@ function redirect($url) {
 $json = getData($url);
 $releases = json_decode($json, TRUE);
 
-$osregex = array("lin" => "/^lin/i", "osx" => "/^osx/i", "win" => "/exe$/i", "wzp" => "/^win.*zip$/i");
+$osregex = array("lin" => "/^lin-/i", "osx" => "/^osx-/i", "win" => "/\.exe$/i", "wzp" => "/^win-.*\.zip$/i", "uzp" => "/^(?!win-|osx-|lin-|wzp-).*\.zip$/i");
```

Comment:
I don't think anyone is using the wzp prefix, but it probably can't hurt to leave it in there. The collision potential is super small.

Reply:
And, either revert the addition of "-" and "." (preferred) or update the docs. Come on, be consistent.

Reply:
I think that "-" and "." are important for regex, will update docs.

Reply:
Why?

Reply:
Because for eg: LinkedQuests.zip filename will incorrectly match 'Linux Package'.